cmd/sebak/cmd/run.go

```diff
@@ -374,18 +374,17 @@ func parseFlagsNode() {
 	queries.Add("IdleTimeout", "3s")
 	bindEndpoint.RawQuery = queries.Encode()
 
-	if len(flagPublishURL) > 0 {
-		if p, err := common.ParseEndpoint(flagPublishURL); err != nil {
-			cmdcommon.PrintFlagsError(nodeCmd, "--publish", err)
-		} else {
-			publishEndpoint = p
-			flagPublishURL = publishEndpoint.String()
-		}
+	// If there is no endpoint, just set as default
```

Comment:
This comment is the best part :)

Reply:
Except for the little typo (set a default, not "as default")